src/model/bpmn/internal/shape/ShapeBpmnEventBasedGatewayKind.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Values available for the `eventGatewayType` property in the BPMN specification.
+ */
+export enum ShapeBpmnEventBasedGatewayKind {
```

Comment:
I would add the type 'None' like we do with the other kind, instead of putting this attribute as Optional.

Reply:
I didn't notice the 'None' entries, I will check and update for consistency.

Reply:
fixed with dc2bada